### ISSUE TITLE
mintty 3.7.2 fails to compile

### ISSUE THREAD

**Issue by Biswa96** [mintty/mintty#1275]
compiler error:

```
appinfo.c:2:10: fatal error: appinfo.t: No such file or directory
    2 | #include "appinfo.t"
      |          ^~~~~~~~~~~
```

This happens with `make -j2` but not with one job.

**Comment by mintty** [comment]
That file is auto-generated via Makefile. Not sure why `-j2` triggers this failure, go without for now.

**Comment by k-takata** [comment]
How about this?
```diff
--- a/src/Makefile
+++ b/src/Makefile
@@ -158,7 +158,7 @@ $(BINDIR)/term.o: term.c emojibase.t emojiseqs.t
 $(BINDIR)/wintext.o: wintext.c
 	$(CC) -c $(DEPOPT) $(CCFLAGS) -O0 $(CPPFLAGS) -fstack-check $< -o $(BINDIR)/wintext.o
 
-$(BINDIR)/appinfo.o: appinfo.?
+$(BINDIR)/appinfo.o: appinfo.c appinfo.h appinfo.t
 	$(CC) -c $(DEPOPT) -O0 $(CPPFLAGS) -fstack-check $< -o $(BINDIR)/appinfo.o
 
 $(BINDIR)/%.o: %.c
@@ -172,6 +172,8 @@ $(BINDIR)/%.o: %.rc %.h %.mft
 
 gitversion := $(shell ./mkvertag "$(srcversion)")
 
+appinfo.t: stamp
+
 stamp:
 	echo "#define VERSION_STAMP \"$(gitversion)\"" > appinfo.n
 	cmp appinfo.n appinfo.t && rm -f appinfo.n || /bin/mv appinfo.n appinfo.t
```

**Comment by k-takata** [comment]
Another workaround would be: `make -C src stamp && make -j2`

**Comment by mintty** [comment]
Thanks, @k-takata.
Actually, 3.7.2 also has a bug in the newly introduced version-stamp mechanism, with the effect that the version info in "About" or `-V` is broken.
(Not sure whether that's a sufficient reason to make a new release.)
As this was about an MSYS2 build, I guess version info will also be weird at least if MSYS2 still applies patches to mintty. If it does, maybe we should try to get rid of them.

**Comment by k-takata** [comment]
The version info is also broken in Cygwin:
```
$ mintty --version
mintty  5046cfa6 (x86_64-pc-cygwin)
```

**Comment by mintty** [comment]
Yeah I know and it bothers me. Yet not sure whether to fix just this with a release.

**Comment by mintty** [comment]
Released 3.7.3.
